docs/data_structures.md

```diff
@@ -0,0 +1,64 @@
+| **Tile Z** | The z coordinate of the tile defines the zoom level. Greater values for z will correspond to higher spatial resolution of the corresponding image. For most regions Bing provides images up to zoom level 18. For aerial imagery or images captured by UAVs even higher z values are valid. |
+| **Tile X**| The x coordinate characterises the longitudinal position of the tile in the overall tile map system taken the zoom level into account. The x coordinates increase from west to east starting at a longitude of -180 degrees. |
+| **Tile Y** | The y coordinate characterises the latitudinal position of the tile in the overall tile map system taken the zoom level into account. The latitude is clipped to range from circa -85 to 85 degrees. The y coordinates increase from north to south starting at a latitude of around 85 degrees. |
+| **Geometry** | Each task has a polygon geometry, which can be generated by its x, y and z coordinates. At the equator the task geometry is a square with an edge length of around 150 metres covering circa 0.0225 square kilometres. Due to the web Mercator projector the task geometry will be clinched with increasing distance to the equator. At the same time the area per task will decrease. |
+| **Tile URL** | The tile URL points to the specific tile image described by the x, y, and z coordinates. Usually, the image has a resolution of 256 x 256 pixels. However, some providers also generate image tiles with higher resolution (e.g. 512 x 512 pixels). |
+
+
+## Results
+Results contain information on the user classifications. However, only “Yes”, “Maybe” and “Bad Imagery” classifications are stored as results. Whenever users indicate “No building” by just swiping to the next set of tasks, no data entry is created. “No Building” classifications can only be modelled retrospectively for groups where a user also submitted at least one “Yes”, “Maybe” or “Bad Imagery” classification.
+
+| Parameter | Description | Values: Built areas | Values: Building footprints |
+| --- | --- | --- | --- |
+| **Id** | Each result can be identified by its Id. The Id is a combination of task Id and user Id. |
+| **Task Id** | Each result corresponds to a specific task, which can be described by its Id. |
+| **User Id** | Each result is contributed by a specific user. Users can be identified by their Id. |
+| **Timestamp** | The timestamp (in milliseconds since 01-01-1970) provides information on the time the user completed the group and uploaded the result data. Results within the same group are assigned the same timestamp. |
+| **Result** | This parameter describes the given answer. Each user can only submit one result per task. | 1: “Yes"<br/>2: “Maybe”<br/>3: “Bad Imagery”. | 1: Footprint looks correct.<br/>2: Footprint is off (wrong size and/or alignment)<br/>3: Bad imagery (impossible to confirm)  |
```

Comment:
using integers for the results sounds good. this will not change from what we already have. However, we might beed to think about another tale etc. where we store what's the meaning of 1,2, 3 for the individual types.

Reply:
this is how results are currently stored in firebase:
```
"18-161584-127536" : {
    "GsyjLsKH9xVbPo0WYf7fCK9irTD2" : {
      "data" : {
        "device" : "c83c6ff14d5e8364",
        "id" : "18-161584-127536",
        "item" : "BUILDINGS",
        "projectId" : "13531",
        "result" : 1,
        "timestamp" : 1539341905490,
        "user" : "GsyjLsKH9xVbPo0WYf7fCK9irTD2",
        "wkt" : ""
      }
    }
  }
```

The `wkt` attribute refers to the geometry in Well-Known-Text format. In my opinion we should try to stick to this format as close as possible. maybe adding some attributes to data, but ideally only a few.

For example I would like to add `groupId` as an attribute.